config/m4/cuda.m4

```diff
@@ -66,6 +66,13 @@ AS_IF([test "x$cuda_checked" != "xyes"],
                                     [AC_MSG_ERROR([libnvidia-ml not found. Install appropriate nvidia-driver package])])
                               cuda_happy="no"])])
 
+         # Check for nvmlDeviceGetGpuFabricInfoV
+         AC_CHECK_DECLS([nvmlDeviceGetGpuFabricInfoV],
+                        [AC_DEFINE([HAVE_NVML_FABRIC_INFO], 1, [Enable NVML GPU fabric info support])],
+                        [AC_MSG_NOTICE([nvmlDeviceGetGpuFabricInfoV function not found in libnvidia-ml. MNNVL support will be disabled.])],
```

Comment:
without that we can't check MNNVL reachability, so I'd say yes we can't support MNNVL properly.

Reply:
so maybe have single macro HAVE_MNNVL that depends on having all required functions including nvmlDeviceGetGpuFabricInfoV\;

Reply:
i think we still want to be able to allocate fabric memory if needed. We have `UCX_CUDA_COPY_ENABLE_FABRIC` for managing that

Reply:
AC_CHECK_DECLS would define a macro like HAVE_DECL__NVML_DEVICE_GET_GOU_FABRIC_INFO_V automatically, maybe just use it? instead of adding HAVE_NVML_FABRIC_INFO